conda/template/conda_build_config.yaml

```diff
@@ -1,19 +0,0 @@
-moose_dev:
```

Comment:
With this file being turned into a symlink, the header information in the relevant `meta.yaml` files that mention modifying both the `moose/conda_build_config.yaml` and this one should be adjusted.